app/Controllers/ReservationsAdminController.php

```
@@ -121,12 +121,14 @@ public function update(Request $request, Response $response, mysqli $db, int $id
         if ($dataFineRichiesta !== '' && !preg_match('/^\d{4}-\d{2}-\d{2}$/', $dataFineRichiesta))
             $dataFineRichiesta = '';
 
-        $today = date('Y-m-d');
+        $today = \App\Support\DateHelper::today();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\DateHelper' in method 'update'. (undefined)

(StaticAccess)